src/components/Button.js

```diff
@@ -1,22 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Styles from '../styles';
 
 const Button = props => {
-  const { btnName } = props;
+  const { btnName, wide, color } = props;
+  const style = {
+    width: wide ? '50%' : '25%',
```

Comment:
[Optional] 
Microverse encourage DRY-(Don't Repeat Yourself) code
I think you could set the wide default value to `25%` i.e `true = 25%`